### PR TITLE
Import automation: finishing Deployment section in the README

### PR DESCRIPTION
**Pull request by intrepiditee**
The section includes a description for /gcbrun.

Thanks for reviewing.

### CODE REVIEW

File: import-automation/README.md [hunk 1]
```diff
@@ -132,4 +132,58 @@ directories:
 
 ## Deployment
 
-TODO(intrepiditee): Finish.
+1. Check in the `import-automation` directory to the repository.
```
Comment (jeffreyoldham) [inline import-automation/README.md:135]: I am not quite sure what this requires.

File: import-automation/README.md [hunk 2]
```diff
@@ -132,4 +132,58 @@ directories:
...
+2. [Configure](executor/README.md#configuring-the-executor) and [deploy](executor/README.md#deploying-on-app-engine) the executor
+3. [Deploy the import progress dashboard API]((import-progress-dashboard-api/README.md#deploying-to-app-engine))
+4. [Deploy the import progress dashboard frontend]((import-progress-dashboard-frontend/README.md#deploying-to-app-engine))
```
Comment (jeffreyoldham) [inline import-automation/README.md:138]:
```suggestion
3. [Deploy the import progress dashboard API](import-progress-dashboard-api/README.md#deploying-to-app-engine)
4. [Deploy the import progress dashboard frontend](import-progress-dashboard-frontend/README.md#deploying-to-app-engine)
```

File: import-automation/README.md [hunk 3]
```diff
@@ -132,4 +132,58 @@ directories:
...
+     - **Event**: `Push to a branch`
+     - **Build Configuration**
+       - **Substitution variables** (keeping the others)
+         - **_BASE_BRANCH**: `master`
+         - **_HEAD_BRANCH**: `master`
+         - **_PR_NUMBER**: `0`
+         - **_HANDLER_URI**: `/schedule`
+
+
+
```
Comment (jeffreyoldham) [inline import-automation/README.md:189]:
```suggestion
```

File: import-automation/README.md (hunk 1 above)
Comment (intrepiditee) [reply import-automation/README.md:135]: just saying the directory needs to be in the repo

**Review by tjann — APPROVED** [review]

**Comment by intrepiditee** [comment]
@tjann @jeffreyoldham This PR should be ready to be merged

**Pull request merged**